### PR TITLE
Delete invisible borders

### PR DESCRIPTION
**Pull request by spisladqo**
Delete these borders, so we can freely move vertices where we want to.

### CODE REVIEW

**Review by shakareem — APPROVED** [review]
The so-called "Big PR"

File: app/src/main/kotlin/viewmodel/graph/VertexViewModel.kt [hunk 1]
```diff
@@ -24,13 +24,9 @@ class VertexViewModel<D>(
         get() = vertex.data.toString()
 
     fun onDrag(dragAmount: DpOffset, currentWindowVM: WindowViewModel, density: Float, scale: Float) {
```
Comment (qrutyy) [inline app/src/main/kotlin/viewmodel/graph/VertexViewModel.kt]: In this case, we don't need the density and scale. I recommend trying to fix the issue of different resolutions. In that case, these input variables might be needed.

TOFIX

**Review by qrutyy — CHANGES_REQUESTED** [review]

File: app/src/main/kotlin/viewmodel/graph/VertexViewModel.kt (hunk 1 above)
Comment (spisladqo) [reply app/src/main/kotlin/viewmodel/graph/VertexViewModel.kt]: There's no issues with different resolutions, but I agree that there's no need in these parameters and they could be thrown away.

Comment (qrutyy) [reply app/src/main/kotlin/viewmodel/graph/VertexViewModel.kt]: GJ

**Review by qrutyy — APPROVED** [review]

**Pull request merged**